ideascube/serveradmin/tests/test_backup.py

```diff
@@ -19,7 +18,7 @@ def test_backup_init_should_raise_with_malformatted_string():
 
 
 def test_underscore_as_separator_should_still_be_supported():
-    Backup('edoardo-0.0.0-201501231405.zip')
+    Backup('edoardo-0.0.0-201501231405.tar.gz')
```

Comment:
Huh?

The method talks about underscores as separator, but I can't see any underscore here?